dashboard/test/helpers/pd/workshop_survey_results_helper_test.rb

```diff
@@ -860,4 +860,188 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
+              'Facilitator Person 2' => {
+                'Sometimes' => 5,
+                'Almost never' => 2
+              }
+            }
+          }
+        },
+        'Day 2' => {
+          general: {
+            'iFeel133' => {
+              'Fantastic' => 5,
+              'Pretty good' => 3
+            },
+            'overallHow' => {
+              'A tremendous amount' => 5
+            }
+          },
+          facilitator: {
+            'howOften56' => {
+              'Facilitator Person 1' => {
+                'All the time' => 4,
+                'Often' => 2
+              },
+              'Facilitator Person 2' => {
+                'Sometimes' => 3,
+                'Almost never' => 5
+              }
+            }
+          }
+        },
+      },
+      all_my_workshops: nil,
+      questions: {
+        'Day 1' => {
+          general: {
+            'overallHow' => {
+              text: 'Overall how much did you learn from this course?',
+              answer_type: 'singleSelect',
+              options: [
+                'Almost nothing',
+                'A little bit',
+                'Some',
+                'Quite a bit',
+                'A tremendous amount'
+              ]
+            }
+          },
+          facilitator: {
+            'howOften56' => {
+              text: 'How often did your facilitator help you?',
+              options: [
+                'Almost never',
+                'Once in a while',
+                'Sometimes',
+                'Often',
+                'All the time'
+              ]
+            }
+          }
+        },
+        'Day 2' => {
+          general: {
+            'overallHow' => {
+              text: 'Overall how much did you learn from this course?',
+              answer_type: 'singleSelect',
+              options: [
+                'Almost nothing',
+                'A little bit',
+                'Some',
+                'Quite a bit',
+                'A tremendous amount'
+              ]
+            },
+            'iFeel133' => {
+              text: 'Overall, how do you feel about computer science?',
+              answer_type: 'singleSelect',
+              options: [
+                'Awful',
+                'Not great',
+                'So-so',
+                'Pretty good',
+                'Fantastic'
+              ]
+            }
+          },
+          facilitator: {
+            'howOften56' => {
+              text: 'How often did your facilitator help you?',
+              options: [
+                'Almost never',
+                'Once in a while',
+                'Sometimes',
+                'Often',
+                'All the time'
+              ]
+            }
+          }
+        }
+      },
+      facilitators: @workshop.facilitators.map {|f| [f.id, f.name]}.to_h
+    }
+
+    expected_averages = {
+      'Facilitator Person 1' => {
+        'overallHow' => {
+          this_workshop: 4.64,
```

Comment:
These are magic numbers; I don't know how they are calculated. Can we make it easier to figure out what is wrong if this test fails in the future? (Assuming the person maintaining the code is different from the author.)

Reply:
We don't have an assertion for not throwing an exception (presumably because if it does throw an exception the test will have an error), so I added a comment and changed the name of the test to better express what it is testing